code/src/components/MovieCard.js

```diff
@@ -0,0 +1,24 @@
+import React from "react"; 
+import { Link } from "react-router-dom"; 
+
+export const MovieCard = ({id, poster_path, original_title, release_date}) => {
+
+  return (
+    <>
```

Comment:
i haven't seen this before. what does this empty tag do? i'm curious. 🤔

Reply:
React fragment :p

Reply:
No need for React fragment over here since you return only one element (`.movie-card`)